yarn-project/end-to-end/src/fixtures/token_utils.ts

```diff
@@ -43,3 +43,21 @@ export async function expectTokenBalance(
   logger.info(`Account ${owner} balance: ${balance}`);
   expect(balance).toBe(expectedBalance);
 }
+
+export async function mintNotes(
+  sender: Wallet,
+  recipient: AztecAddress,
+  asset: TokenContract,
+  noteAmounts: bigint[],
+): Promise<bigint> {
+  // We can only mint 4 notes at a time, since that's the maximum number of calls our entrypoints allow
```

Comment:
```suggestion
  // We can only mint 4 notes at a time, since that's the maximum number of calls our entrypoints allow
  // TODO(#13024): mint as many notes as possible in a single tx
```